pkgs/servers/web-apps/peering-manager/default.nix

```diff
@@ -44,7 +20,7 @@ in py.pkgs.buildPythonApplication rec {
 
   format = "other";
 
-  propagatedBuildInputs = with py.pkgs; [
```

Comment:
The `py → python` change is unrelated to that commit, can we separate it perhaps?